sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -784,7 +785,8 @@ class DataSourceV2SQLSuite
       sql("SHOW NAMESPACES")
     }
 
-    assert(exception.getMessage.contains("No default v2 catalog is set"))
+    assert(exception.getMessage.contains(
+      "SHOW NAMESPACES is not supported with the session catalog"))
```

Comment:
Why is this not supported?

Reply:
This is the existing behavior. We will support it once we unify SHOW NAMESPACE and SHOW DATABASE. AFAIK @imback82 already has a PR for it.

Reply:
Yes, I will update this as soon as this PR is merged.